README.md

```diff
@@ -28,6 +28,17 @@ gulp.src("./src/**/hello.txt")
   }))
   .pipe(gulp.dest("./dist")); // ./dist/main/text/ciao/hello-goodbye.md
 
+// rename via function's return value
+gulp.src("./src/**/hello.txt")
+  .pipe(rename(function (path) {
+    return {
```

Comment:
Is this supposed to operate as a merge or a full map? Might be a good example to put a comment here so people know to return the full object.

Reply:
Makes sense. I'll put a comment indicating that the old value will be discarded and overwritten.